below/render/src/default_configs.rs

```diff
@@ -275,7 +275,27 @@ impl HasRenderConfig for model::NetworkModel {
     }
 }
 
-impl HasRenderConfigForDump for model::NetworkModel {}
+impl HasRenderConfigForDump for model::NetworkModel {
+    fn get_openmetrics_config_for_dump(
+        &self,
+        field_id: &Self::FieldId,
+    ) -> Option<RenderOpenMetricsConfigBuilder> {
+        use model::NetworkModelFieldId::*;
+        match field_id {
+            // Do not dump interfaces from network model b/c there is already the `iface` category.
+            // It is also hard to figure out exactly which interface is being queried from here.
+            // Meaning we cannot label the metric with the proper interface.
+            Interfaces(_) => None,
```

Comment:
We could probably add a ToString trait bound on FieldId to make this doable, but no big deal